omnibus/files/private-chef-ctl-commands/global_admins.rb

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'restclient'
+require 'json'
+require 'sequel'
+
+PLACEHOLDER_GLOBAL_ORG_ID = "00000000000000000000000000000000"
+
+add_command_under_category "grant-global-admin-permissions", "global-admins", "Grant a user the ability to create other users by added the user to the global-admins group.", 2 do
+
+  cmd_args = ARGV[3..-1]
+  if cmd_args.length != 1
+    msg = "Username is the only argument to grant-global-admin-permissions.\nPlease pass a single argument."
+    STDERR.puts msg
+    raise SystemExit.new(1, msg)
+  end
+
+  username = cmd_args[0]
+
+  db = setup_erchef_db
+  global_admins_authz_id = get_global_admins_authz_id(db)
+  user_authz_id = get_user_authz_id(db, username)
+
+  # put the user in the global-admins authz group
+  headers = {
+    :content_type => :json,
+    :accept => :json,
+    'X-Ops-Requesting-Actor-Id' => running_config['private_chef']['oc_bifrost']['superuser_id']
```

Comment:
Two things - a new function `running_service_config(servicename)` has been added and ready for use once the LOB-10-bugfix PR lands shortly in omnibus-ctl. 

I'd also like to see us just grab it once, `bifrost_config = running_service_config('oc_bifrost')`, etc.